dm/pkg/schema/tracker.go

```diff
@@ -509,7 +529,7 @@ func (tr *Tracker) BatchCreateTableIfNotExist(tablesToCreate map[string]map[stri
 			cloneTis = append(cloneTis, cloneTi)
 		}
 		schemaName := model.NewCIStr(schema)
-		if err := tr.dom.DDL().BatchCreateTableWithInfo(tr.se, schemaName, cloneTis, ddl.OnExistIgnore); err != nil {
+		if err := tr.SplitBatchCreateTableAndHandle(schemaName, cloneTis, 0, len(cloneTis)); err != nil {
```

Comment:
I'm a bit nervous about zero length cloneTis. Do we have a test for it?

Reply:
Let me try. If error occurs I would just skip this operation.

Reply:
I check the codes and think there is no case with zero length cloneTis. For safety, I add a test in ``schema_test.go``.